pkgs/applications/networking/p2p/transmission/4.nix

```diff
@@ -94,6 +96,7 @@ stdenv.mkDerivation (finalAttrs: {
   ]
   ++ lib.optionals enableGTK3 [ wrapGAppsHook ]
   ++ lib.optionals enableQt [ qt5.wrapQtAppsHook ]
+  ++ lib.optionals stdenv.isDarwin [ libicns writeDarwinBundle ]
```

Comment:
@emilytrau could it be the `libicns` addition here should be part of the previous `fix darwin build` commit?